soroban-env-common/env.json

```diff
@@ -1422,6 +1422,58 @@
                 },
                 {
                     "export": "a",
+                    "name": "extend_current_contract_instance_and_code_ttl_separately",
```

Comment:
We can't modify the host function signatures, only new functions can be added. They also need a protocol guard.

Reply:
Ah this is the feedback I was looking for, makes sense thanks. Is there an interface reference for setting the protocol guard for the function?

Reply:
Protocol gating has just been merged: https://github.com/stellar/rs-soroban-env/pull/1348 . Just follow the example of `protocol_gated_dummy`, but use protocol 21 for min supported version.